src/cosmology/api/_extras.py

```diff
@@ -55,16 +91,13 @@ def H(self, z: InputT_contra, /) -> ArrayT_co:
         """  # noqa: D402
         ...
 
-    def h_over_h0(self, z: InputT_contra, /) -> ArrayT_co:
```

Comment:
Has good old `Efunc` left us for good?

Reply:
I think so. We never could figure out a good name for the method besides this one.